.github/workflows/GHPages.yml

```diff
@@ -19,9 +34,12 @@ jobs:
           yarn install
           yarn pre-build
           yarn build
-      - name: Deploy
-        uses: peaceiris/actions-gh-pages@v3
+      - name: Setup Pages
+        uses: actions/configure-pages@v1
+      - name: Upload artifact
+        uses: actions/upload-pages-artifact@v1
         with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          publish_dir: ./explorer-v2/build
-          force_orphan: true
+          path: ./explorer-v2/build
+      - name: Deploy to GitHub Pages
```

Comment:
Will `.nojekyll` be added automatically?

Reply:
I don't think this workflow uses jekyll.

Example (probably) without jekyll:
https://github.com/actions/starter-workflows/blob/main/pages/static.yml

Example using jekyll:
https://github.com/actions/starter-workflows/blob/main/pages/jekyll.yml

Reply:
Thanks seems great!